css-brochure-form/styles.css

```diff
@@ -0,0 +1,71 @@
+#engineers {
+  margin-bottom: 15px;
+  width: 400px;
+}
+
+#where {
+  width: 400px;
+}
+
+#name {
+  width: 400px;
+}
+
+#email {
+  width: 400px;
+}
```

Comment:
Rather than having 4 different CSS rulesets that are specifically targeting each element but applying the same styling, how could we create one ruleset to apply to all of these form elements?